bitcoin/src/blockdata/transaction.rs

```diff
@@ -1170,10 +1170,10 @@ impl From<&Transaction> for Wtxid {
 
 /// Computes the value of an output accounting for the cost of spending it.
 ///
-/// The effective value is the value of an output value minus the amount to spend it.  That is, the
-/// effective_value can be calculated as: value - (fee_rate * weight).
+/// The effective value is the value of an output minus the cost to spend it.  That is, the
```

Comment:
Shouldn't it say "cost of spending it"?

Reply:
Both are grammatically correct. "of" goes with "spending" and "to" goes with "spend" - English, I know.

Reply:
"spending" somehow feels more natural here but I don't know why. But I don't really care unless there's an actual difference in meaning.